cmd/kyma/install/cmd.go

```diff
@@ -40,7 +38,7 @@ func NewCmd(o *Options) *cobra.Command {
 		Use:   "install",
 		Short: "Installs Kyma on a running Kubernetes cluster.",
 		Long: `Use this command to install Kyma on a running Kubernetes cluster.
-
+		
 ### Detailed description
```

Comment:
Do we have "vague description" too? What does "detailed" mean, is this word really needed?